devconf_us_aug_2024/devconf_us_aug_2024.md

```diff
@@ -0,0 +1,166 @@
+
+Finally, Kelsey reminded us that [documentation](https://www.youtube.com/live/dWCxGZix75s?si=zzOdAz2kseUi1d2f&t=1513) is an essential part of the engineering process.
+
+
+### [Keynote: Operation and AI with Jen Krieger](https://www.youtube.com/watch?v=MZrfs9Rswew)
+
+_Quote:_
+**Become a system thinker in everything you do; and be kind** (Jen Krieger)
+
+#### Takeaways
+- **Change lead time** plays a critical role in how organizations perform and adapt to business challenges.
+- Similar to previous waves of automation, AI will impact the IT profession by rendering some jobs obsolete while creating and transforming others.
+- Some technology professions cannot fully automate their tasks, sometimes leading to bottlenecks in automated environments. 
+  AI tools can help address these bottlenecks by automating specific tasks.
+- To avoid frustration, collaboration and transparency between business and engineering are essential.
+- Humans remain at the core of an enterprise's success.
+
+The second day began with a keynote by [Jen Krieger](https://x.com/mrry550?lang=en). 
+Jen Krieger is a tech enthusiast with 25 years of experience in the open-source software space and is currently the VP of Engineering Operations at GitHub.
+
+Jen spoke about AI and its impact on how we work. AI has existed for some time, but recent advancements have brought its usage to a broader audience.
+Jen highlights the fact that we have a conflicted relationship with **change**. We need to remain open to evaluating the new tools introduced by the 
+current AI movement and embrace them to improve our productivity when they prove useful. Technology can be both a benefit to our personal lives and 
+a potential threat to our jobs. The current AI trend is similar to past groundbreaking technologies. We need to be mindful of the **social changes** 
+brought about by technology and adopt a **systems thinking** approach to its implementation.
+
+Some technology professions cannot fully automate their tasks. For instance, **product management** tools have not evolved as quickly as development tools. 
+This can create bottlenecks in delivering enterprise software, potentially leading to unsatisfied customers and lost revenue.
+AI tools can assist **product managers** by analyzing large datasets, automating workflows, and enhancing decision-making capabilities.
+
+Jen highlights the importance of engineering and business teams collaborating to understand each other's processes and agreeing on shared workflows to streamline software delivery.
+Some companies have achieved a change lead time of less than one day. High-performing teams are often the result of investments in development processes. 
+Agile practices, DevOps transformations, and Cloud-Native development are a few examples of processes that help streamline the software development lifecycle.
+
+Finally Jen emphasizes that companies need to empower teams and invest in upskilling. 
```

Comment:
You use past tense and present tense within the summary. 
Eg: "Jen spoke about AI.." vs "Jen highlights the importance of engineering.."

The talk with Kelsey Hightower is written in past tense but the other talks are written in present tense. I would suggest sticking to one tense throughout the article.

Reply:
Fixed :)